### DISCUSSION TITLE
Welcome to Iris Discussions!

### DISCUSSION THREAD

**Discussion by coderbot16** [IrisShaders/Iris#23] [General]
## 👋 Welcome!

I've enabled the Discussions feature as a way for community members to communicate without relying on Discord. Discussions are more open-ended than issues and hopefully provide an opportunity to have some more casual conversation!

Discussions are good for:

* Asking questions about Iris and its development
* Sharing ideas
* Engaging with other community members

Issues are good for:

* Reporting crashes and other bugs in Iris

**Comment by IMS212** [comment]
nice

**Comment by YTG1234** [comment]
hello

**Reply by BeansBeefBroccoli** [reply]
hi 😄

**Comment by allnought** [comment]
Now this is epic.

**Comment by ghost** [comment]
Is there any way to run custom chaders as BSL or Chocapic13's UltraPerfomance right now? 
How to do that?

**Reply by IMS212** [reply]
No, you cannot. Best thing that's running right now is Depth of Field EDIT: Iris can now run Sildurs Vibrant, and Enhanced default to an extent .

**Comment by BeansBeefBroccoli** [comment]
I'm sorry I'm not very familiar with GitHub so please don't judge me, but where do I download Iris, because it wasn't in Releases (probably because its not a full release) but idk how to get anywhere else.

**Reply by Yales21** [reply]
i think you can't download a  iris release on github, you have to build your own using the gradlew.bat on windows or the ./gradlew build on linux and macos. Then the files will be in /Iris/build/libs/.

**Reply by BeansBeefBroccoli** [reply]
ok thanks so much

**Reply by IMS212** [reply]
Yes, this is correct, but just so you know you cannot use most shaders yet.

**Reply by Altheran88** [reply]
@IMS212 Ao, let say I'd like to run SEUS on Iris, it wouldn't run ? Right now, I built the Source into 4 .jar files. I'vs put  iris-0.1.0.jar into my mods folder. I've extracted the SEUS-Renewed.Zip as a Folder under the /shaderpacks folder.

In game, I I press F3, I see that "Iris (internal Shaders) is running, But I have no option to select a shaderpack. So, how do I use a shaderpack ?

**Reply by IMS212** [reply]
As of right now the only big shader working is sildur's vibrant, seus causes a black screen. To select a shaderpack you need to change config/iris.properties.

**Comment by Titaniumtown** [comment]
hello

**Reply by BeansBeefBroccoli** [reply]
hi :)

**Reply by OverlordsIII** [reply]
hoi

**Reply by BeansBeefBroccoli** [reply]
hoi

**Comment by BeansBeefBroccoli** [comment]
When people keep saying "Sildurs" do they mean only Sildurs Vibrant Shaders, or Sildurs Enhanced Default too?

**Reply by Titaniumtown** [reply]
idk m8

**Reply by OverlordsIII** [reply]
I think it's Vibrant

**Reply by BeansBeefBroccoli** [reply]
ok thanks

**Comment by zhuher** [comment]
SEUS PTGI please b0ss

**Comment by Androkai** [comment]
I just stumbled across the project by chance and just wanted to thank you for the really excellent description in the readme.
